Makefile

```diff
@@ -12,18 +12,18 @@ GIT_SAMPLE_DATA_REPO        := https://bitbucket.org/natcap/invest-sample-data.g
 GIT_SAMPLE_DATA_REPO_PATH   := invest-sample-data
 GIT_SAMPLE_DATA_REPO_REV    := c8df675a2c446bf8d00ffd8f0cbab933f7d5c25a
 
-.PHONY: help clean html changes linkcheck
+.PHONY: help clean html changes linkcheck prep_sampledata test_investspec demo_investspec
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
-	@echo "  clean           to remove the build directory"
-	@echo "  html            to make standalone HTML files"
-	@echo "  changes         to make an overview of all changed/added/deprecated items"
-	@echo "  linkcheck       to check all external links for integrity"
-	@echo "  get_sampledata  to check out the invest-sample-data repo"
-	@echo "  prep_sampledata to create modified tables in invest-sample-data that display nicely"
-	@echo "  test_investspec to run unit tests for the custom Sphinx extension"
-	@echo "  demo_investspec to run a demo using the custom Sphinx extension"
+	@echo "  clean               to remove the build directory"
+	@echo "  html                to make standalone HTML files"
+	@echo "  changes             to make an overview of all changed/added/deprecated items"
+	@echo "  linkcheck           to check all external links for integrity"
+	@echo "  invest-sample-data  to check out the invest-sample-data repo"
```

Comment:
It looks like this is no longer a target

Reply:
It is, as `$(GIT_SAMPLE_DATA_REPO_PATH)`, but maybe filepath targets don't belong in the help section, not sure

Reply:
You could clarify this by using `@echo "   $(GIT_SAMPLE_DATA_REPO_PATH) ...` in `make help` ... I think that would render correctly!

Reply:
Thanks for the clarification. James' idea seems like a good one

Reply:
Good idea! It does kind of throw off the spacing in the Makefile, since you have to adjust for the length difference between `$(GIT_SAMPLE_DATA_REPO_PATH)` and `invest-sample-data`, but that's not a problem.